### PR TITLE
[Wallet]: update wallet image

### PR DESCRIPTION
**Pull request by plam-ml**
## Description 

- Update wallet app images from circle to square
- https://mysten.atlassian.net/browse/APPS-1535

Screenshot 2023-09-06 at 1 16 15 PM

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/ui/app/shared/image-icon/index.tsx [hunk 1]
```diff
@@ -10,17 +10,22 @@ const imageStyle = cva(['text-white capitalize overflow-hidden bg-gray-40  shrin
...
 		},
+		background: {
+			none: 'bg-transparent',
+		},
 	},
 
 	defaultVariants: {
 		circle: false,
 		size: 'md',
+		background: 'none',
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/shared/image-icon/index.tsx:28]: Does this even need to be a variant? Could you just make this part of the default classes?

File: apps/wallet/src/ui/app/shared/image-icon/index.tsx [hunk 2]
```diff
@@ -10,17 +10,22 @@ const imageStyle = cva(['text-white capitalize overflow-hidden bg-gray-40  shrin
 			sm: 'w-6 h-6 font-medium text-subtitleSmallExtra',
 			md: 'w-7.5 h-7.5 font-medium text-body',
 			lg: 'w-10 h-10 font-medium text-heading4',
-			xl: 'w-15 h-15 font-medium text-heading4',
+			xl: 'w-12.5 h-12.5 font-medium text-heading4',
+			xxl: 'w-15 h-15 font-medium text-heading4',
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/shared/image-icon/index.tsx:14]: Does this xxl variant actually get used?

Comment (plam-ml) [reply apps/wallet/src/ui/app/shared/image-icon/index.tsx]: Yea, I just updated it on subsequent pushes

File: apps/wallet/src/ui/app/shared/image-icon/index.tsx (hunk 1 above)
Comment (plam-ml) [reply apps/wallet/src/ui/app/shared/image-icon/index.tsx]: updated

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**